udata_front/theme/gouvfr/assets/js/components/DataservicesSearch/DataservicesSearch.vue

```diff
@@ -180,7 +180,7 @@ const { ignoreUpdates } = watchIgnorable(params, () => {
   window.history.pushState(null, "", url);
 });
 
-const url = computed(() => `/dataservices?${params.value}`)
+const url = computed(() => `/dataservices/search/?${params.value}`)
```

Comment:
Not introduced on this PR but I think we don't drop the page arg when changing other param queries (filter or `q`)

Reply:
Done